src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -463,7 +499,7 @@ export type Config = {
   gravatarBaseUrl: string;
   messages: string[];
   dsn: string;
-  userIdentity: {ip_address: string; email: string; id: number; isStaff: boolean};
+  userIdentity: {ip_address: string; email: string; id: string; isStaff: boolean};
```

Comment:
the Sentry SDK expects id to be a string.

I think it _does_ come back as a number, so this might be slightly wrong, but this is probably OKAY for now